### PR TITLE
Redesign Text Analyzer Flow using MD Bootstrap

### PR DESCRIPTION
**Pull request by chrismohrii**
- Switches over to MDBoostrap elements
- Hides the advanced options for input
- Replaces tooltips with a sticky (floating-ish) sidebar
Video demo: https://drive.google.com/file/d/1POccoxt9QMkgmhbS--P3hnPK1qEZfA8B/preview

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -141,24 +144,58 @@ function addSubstring(substring, analysisContainer, result, loadingEl, response,
...
+    substringEl.appendChild(createAnyElement('span', substring));
   }
   colorSubstring(substringEl, toxicityScore);
-     
-  // Attach a tooltip (info-box for the substring)
-  const tooltipEl = createTooltip(attributes);
-  substringEl.appendChild(tooltipEl);
+
+  // Set up analysis replacement function call after user clicks on segment
+  substringEl.onclick = function() { handleSegmentClick(attributes, substringEl, wordElts, substring, lang);  }
```
Comment (danieldortega77) [inline capstone/src/main/webapp/script.js]: nit:
```suggestion
  substringEl.onclick = function() { handleSegmentClick(attributes, substringEl, wordElts, substring, lang); }
```

File: capstone/src/main/webapp/cover.css [hunk 2]
```diff
@@ -1,7 +1,6 @@
 /* Base structure */
 html, body {
   height: 100%;
-  background-image: url(/assets/Pattern-Randomized.svg);
```
Comment (danieldortega77) [inline capstone/src/main/webapp/cover.css:4]: Just out of curiosity, what happens if we keep this in? The white background might be a bit bland

Comment (chrismohrii) [reply capstone/src/main/webapp/cover.css:4]: It'll look like this. I was going for the style on support.perspectiveapi.com, but happy to meet about style preferences
image

Comment (danieldortega77) [reply capstone/src/main/webapp/cover.css:4]: Ah, I see. I think the no-background looks better now, but maybe we can make horizontal divisions of light gray like support.perspectiveapi.com at some point.

**Review by danieldortega77 — APPROVED** [review]

**Review by olu144 — APPROVED** [review]

**Review by dborkan — APPROVED** [review]
LGTM

File: capstone/src/main/webapp/script.js [hunk 3]
```diff
@@ -136,31 +139,65 @@ function addSubstring(substring, analysisContainer, result, loadingEl, response,
...
   const attributes = sortAttributes(response);
 
-  // Set up replacements function call and color the substring appropriately	
+  // Break up and color the segment appropriately	
   const substringEl = document.createElement('span');
+  const wordElts = [];
   const words = getSubstrings(substring, TokenizerEnum['WORD']);
-  for (let i = 0; i < words.length; i++) {
+  if (words) {
+    for (let i = 0; i < words.length; i++) {
```
Comment (dslucas) [inline capstone/src/main/webapp/script.js:147]: nit: Let's fix the indentation for this for loop and the code within it.

**Review by dslucas — APPROVED** [review]

**Pull request merged**